jetty-maven-plugin/src/it/javax-annotation-api/src/main/resources/log4j.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?><!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
```

Comment:
And this log4j.xml just makes the output appear on the console (something that the it tests expect)